app/services/payment_providers/stripe/handle_event_service.rb

```diff
@@ -89,6 +82,33 @@ def payment_service_klass(event)
           raise NameError, "Invalid lago_payable_type: #{payable_type}"
         end
       end
+
+      def update_payment_status(status)
```

Comment:
I think we should move the logic in dedicated services:
- `PaymentProviders:Stripe::Webhooks::PaymentIntentSucceededService`
- `PaymentProviders:Stripe::Webhooks::PaymentIntentFailedService`

This service should just route the events to the right handler